app/scripts/modules/kubernetes/securityGroup/configure/wizard/upsert.controller.js

```diff
@@ -181,6 +182,16 @@ module.exports = angular.module('spinnaker.securityGroup.kubernetes.create.contr
             cloudProvider: 'kubernetes',
             region: $scope.securityGroup.namespace,
           };
+
+          // Write TLS property so clouddriver understands it and our certs get configured
+          $scope.securityGroup.tls = [];
+          for (var idx in $scope.securityGroup.rules) {
```

Comment:
Please use `let` here instead of `var`. Variables defined with`var` are not block scoped - i.e., `idx` is defined for the remainder of the function after this loop.